EventsExpress.Core/NotificationHandlers/EventCreatedHandler.cs

```diff
@@ -25,18 +28,22 @@ public class EventCreatedHandler : INotificationHandler<EventCreatedMessage>
 
         public async Task Handle(EventCreatedMessage notification, CancellationToken cancellationToken)
         {
-            var users = _userService.GetUsersByCategories(notification.Event.Categories);
             try
             {
-                foreach (var u in users)
+                var userIds = _userService.GetUsersByCategories(notification.Event.Categories).Select(x => x.Id).AsEnumerable();
```

Comment:
Don't do the AsEnumerable here. The return of Select is already IEnumerable. The same applies to below.